### PR TITLE
staging code

### PR DESCRIPTION
**Pull request by olajhidey**
Change of Package Name and added some other information for the pom.xml file
Made some changes also on the README.md file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,100 @@
-# compatibility-api-java
\ No newline at end of file
+# compatibility-api-java
+
+An Unofficial Java API wrapper that allows for the rapid development and integration of Java-based applications with the SignalWire
+API.
+
+signalwire-java automates the process of consuming the Signalwire API by providing a wide range of helper methods that
+allow for quick development and testing.
```
Comment (danieleds) [inline README.md]:
```suggestion
# Java Wrapper for SignalWire Compatibility APIs

An Unofficial Java API wrapper that allows for the rapid development and integration of Java-based applications with the SignalWire
API.

> ⚠️ Disclaimer:
> 
> The libraries in this repository are NOT supported by SignalWire. SignalWire will _not_ answer support questions about the code in this repository, and is not responsible for fixing issues.

This library automates the process of consuming the Signalwire API by providing a wide range of helper methods that
allow for quick development and testing.
```

File: src/test/java/io/github/signalwirecommunity/XMLTest.java [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+    }
+
+    @Test
+    public void getXMLBySIDTest(){
+//        Bin expected = AppTest.getClient().xml().get("xml-sid");
+//        String actual = "xml-friendly-name";
+//        assertThat(expected.name, equalTo(actual));
+    }
+
```
Comment (danieleds) [inline src/test/java/io/github/signalwirecommunity/XMLTest.java]: could you either enable these tests or remove them?

**Review by danieleds — COMMENTED** [review]
could you check that all the dependencies are actually needed? For example what is `unirest-objectmapper-jackson` used for?

File: README.md [hunk 3]
```diff
@@ -1 +1,104 @@
...
+
+This class give you access to the `voice`, `message`, `accounts`, `phonenumber` repository
+
+Get your project Id and API token from your signalWire space. for more information
+click [here](https://developer.signalwire.com/apis/docs/navigating-your-space)
+
+### Sending a Message
+
+```java
+    NewMessageResponse response=client.message().create(to,from,body);
```
Comment (danieleds) [inline README.md]: Could you make sure that all the code blocks in the README are properly indented to the left?

I also suggest a space before and after the ` = ` for all code blocks

File: src/test/java/io/github/signalwirecommunity/XMLTest.java (hunk 2 above)
Comment (olajhidey) [reply src/test/java/io/github/signalwirecommunity/XMLTest.java]: i will probably remove them. I was doing some testing

File: README.md (hunk 3 above)
Comment (olajhidey) [reply README.md]: I will also make this change

**Comment by olajhidey** [comment]
Added fix

**Review by danieleds — APPROVED** [review]

**Pull request merged**